### DISCUSSION TITLE
ESP32+ILI9341 problem with TFT_RST=-1

### DISCUSSION THREAD

**Discussion by andrey-belokon** [Bodmer/TFT_eSPI#1957] [Q&A - General] [answered]
Hi

When reset pin of ILI9341 connected to some ESP32 pin and it defined in TFT_RST all working as expected. But when I try to set TFT_RST to -1 nothing to display. I was try to connect reset pin of ILI9341 to +3.3v, to EN pin of ESP32 board, try RC 10k+0.1u - same result - empty screen.
Is it possible to use ILI9341+ESP32 with TFT_RST=-1 ? For me, this is critical because there are not enough ESP2 pins.

Thank you

**Comment by TangerineDreamer** [comment]
Hi andrey,
i looked at my user setup and see that:

#define TFT_RST  -1  // Set TFT_RST to -1 if display RESET is connected to ESP32 board RST
maybe you should try with that :)

**Reply by andrey-belokon** [reply]
Well, but where is RST on the ESP32 board? I can't find it.

**Reply by Frank-Bemelman** [reply]
You could try the EN pin. Never tried myself..
Or a classic power-on-reset ciruit. 
![71691F37-25D5-43D0-9F59-9D494EA2F769](https://user-images.githubusercontent.com/58854621/184553238-e8f257e6-52c0-45d9-b040-900d8535abf8.jpeg)

**Comment by TangerineDreamer** [comment, marked as answer]
Sorry for my bad, i reproduced silly things written,
I think  : #define TFT_RST -1 means RST goes to 3,3V,
sure TFT_RST=-1 should be written  TFT_RST -1
